workflows/add_scsb_location.md

```diff
@@ -35,16 +35,31 @@ Note the following is a targetted distillation of information in [Discovery Prop
 * **ReCAP customer code**: This uniquely identifies the location in ReCAP.
 * **Allow sierra hold (nypl:allowSierraHold)**: Boolean that should be `true` for any holding location whose items are requestable via native Sierra interfaces. Will be `false` (or unset) for ReCAP locations. (I believe this property is maintained as a formality; The mechanism for showing/suppressing locations lies elsewhere. It appears to often be wrong, e.g. many Scholar rooms delivery locations have it marked as `true`)
```

Comment:
can you also add `deliverableToResolution` to this list?

Reply:
Added